practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1550 @@
+{
```

Comment:
<p>There's an image in here that still does not show up.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
i think @amritpurshotam may have to resolve this. may be the image path is wrong.